content/10.diagnostics.md

```diff
@@ -81,11 +91,18 @@ The authors are now working to get the assay into clinical use [@url:https://www
 
 Importantly, false-positives can occur due to the cross-reactivity with other antibodies according to the clinical condition of the patient [@url:https://www.fda.gov/media/136625/download].
 Therefore, this test should be used in combination with RNA detection tests [@url:https://www.fda.gov/media/136625/download].
+Serological tests also require waiting until a patient has had an imune system reaction sufficient to detect, which means that these tests are of limited use for diagnosis in the early stages of an infection.
```

Comment:
"Due to the long incubation times and delayed immune responses of infected patients, serological tests are insensitive for diagnosis in the early stages of an infection."